src/Compilers/CSharp/Test/Syntax/Resources.resx

```diff
@@ -1,6 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why is this removed?

Reply:
i used the VS resx editor.  it looks like it removes comments.